mutation/mutation_fragment_v2.hh

```diff
@@ -370,6 +370,31 @@ private:
 
 std::ostream& operator<<(std::ostream&, mutation_fragment_v2::kind);
 
+template <> struct fmt::formatter<mutation_fragment_v2::kind> : fmt::formatter<std::string_view> {
+    template <typename FormatContext>
+    auto format(const mutation_fragment_v2::kind& k, FormatContext& ctx) const {
+        string_view name = "UNEXPECTED";
+        switch(k) {
+        case mutation_fragment_v2::kind::static_row:
```

Comment:
nit, could `using enum mutation_fragment_v2::kind`. less typings this way.

Reply:
I already typed it :-) Also, while I was developing this patch I got thoroughly confused between mutation_fragement and mutation_fragment_v2, so it's a good idea to list the full name explicitly.